### PR TITLE
Build-2.9.1

### PR DESCRIPTION
**Pull request by bnolens**

### CODE REVIEW

File: Mammoth/Screens/HomeScreen/NewsFeedViewController.swift [hunk 1]
```diff
@@ -733,6 +731,8 @@ extension NewsFeedViewController {
     }
     
     func scrollViewDidScroll(_ scrollView: UIScrollView) {
+
+        self.isScrollingToTop = !self.tableView.isDecelerating && !self.tableView.isTracking && !self.tableView.visibleCells.isEmpty
```
Comment (rileyhCode) [inline Mammoth/Screens/HomeScreen/NewsFeedViewController.swift:735]: Won't self.isScrollingToTop be set to true (incorrectly) if the user just scrolls a bit and stops (not at the top)?

Comment (bnolens) [reply Mammoth/Screens/HomeScreen/NewsFeedViewController.swift:735]: The variable tracks when the scrolling happens programmatically. I was thinking of a better name but couldn't find one.

Comment (bnolens) [reply Mammoth/Screens/HomeScreen/NewsFeedViewController.swift]: I renamed the variable to `isScrollingProgrammatically`

**Comment by bnolens** [comment]
I'm going to merge this to main so I can make a l10n build including this hotfix. I you have other remarks I'll create a separate PR to address them.

File: Mammoth/Screens/HomeScreen/NewsFeedViewModel+Caching.swift [hunk 2]
```diff
@@ -120,8 +120,8 @@ extension NewsFeedViewModel {
             let scrollPositionCardIndex = items?.firstIndex(where: { $0 == position.model })
 
             if let scrollPositionCardIndex {
-                // Keep the bookmarked card, 8 younger cards, and 10 older cards
-                var cardsSubset = items?[max(scrollPositionCardIndex - 8, 0)...min(scrollPositionCardIndex + 10, (items?.count ?? 1) - 1)]
+                // Keep the bookmarked card, 4 younger cards, and 10 older cards
+                var cardsSubset = items?[max(scrollPositionCardIndex - 2, 0)...min(scrollPositionCardIndex + 10, (items?.count ?? 1) - 1)]
```
Comment (rileyhCode) [inline Mammoth/Screens/HomeScreen/NewsFeedViewModel+Caching.swift:124]: Comment should match the actual number (2).

**Review by rileyhCode — APPROVED** [review]
LGTM. Just one more note about a comment.

**Pull request merged**